### ISSUE TITLE
oxidized-script (0.6.0) gem hardcoded to depend on older oxidized version?

### ISSUE THREAD

**Issue by jp-asdf** [ytti/oxidized-script#36]
(Ubuntu18.04.4 LTS)
I upgraded my oxidized gem from .26 to .27 and removed the old .26 gem with 'gem cleanup'. However oxidized-script (0.6.0) then broke with dependency errors.
I uninstalled the oxidized-script gem, then re-installed with 'gem install oxidized-script'. But every time I reinstall oxidized-script, it also installs the old .26 oxidized even tho I already have .27 installed. Doesn't seem to be checking if something newer is already installed, just wants to install .26.

If you list models in oxs its showing only .26 model paths although I have .27 installed:
```
oxidized@netnms01:~$ gem list | grep oxid
oxidized (0.27.0, 0.26.3)
oxidized-script (0.6.0)
oxidized-web (0.13.1)

oxidized@netnms01:~$ oxs --list-models
         br6910 - /var/lib/gems/2.5.0/gems/oxidized-0.26.3/lib/oxidized/model/br6910.rb
          airos - /var/lib/gems/2.5.0/gems/oxidized-0.26.3/lib/oxidized/model/airos.rb
...
```
Is there a way to make oxidized-script use the deps in oxidized .27? I'd like to try the new linuxgeneric model in a oxs command.

**Comment by ytti** [comment]
I should fix this, I'd need to upgrade 'slop' to newer version in oxidized-script. Oxidized uses newer 'slop' and we can't have two different versions of same package running at the same time.

**Comment by candlerb** [comment]
I see this too.  To reproduce in fresh Ubuntu 20.04 container:

```
# apt-get install ruby ruby-dev build-essential libsqlite3-dev libssl-dev pkg-config cmake libssh-dev libicu-dev bsd-mailx
...

# gem install oxidized --no-document
...
# gem list | grep oxidized
oxidized (0.28.0)

# gem install oxidized-web --no-document
...
# gem list | grep oxidized
oxidized (0.28.0)
oxidized-web (0.13.1)

# gem install oxidized-script --no-document
...
# gem list | grep oxidized
oxidized (0.28.0, 0.26.3)
                  ^^^^^^
oxidized-script (0.6.0)
oxidized-web (0.13.1)
```

**Comment by sajiby3k** [comment]
The issue is oxidized-script uses slop 3; but new oxidized uses slop 4 ruby package.

Some codes in cli.rb in oxidized-script needs to change so that it runs with slop4 because slop3 in not compatible with slop4.

Anyone with slop4 coding experience????

**Comment by ytti** [comment]
Closed as requested by @mortzu

**Comment by candlerb** [comment]
Ref: appears to have been fixed in commit 8e0bbab5
